scripts/generate_fake_driver.sh

```diff
@@ -36,6 +36,9 @@ declare -A files_metadata=(
     ["fully_charged_matrix_effect_static"]="w;"
     ["fully_charged_matrix_effect_wave"]="w;"
     ["game_led_state"]="rw;0"
+    ["hyperpolling_wireless_dongle_indicator_led_mode"]="w;0"
```

Comment:
if it's write-only, no need to give it a default value, so make it like the two items below

Reply:
Done!